lib/tooling/pvs-studio-tool.js

```diff
@@ -33,7 +33,9 @@ import * as utils from '../utils';
 import { BaseTool } from './base-tool';
 
 export class PvsStudioTool extends BaseTool {
-    static get key() { return 'pvs-studio-tool'; }
+    static get key() {
```

Comment:
your IDE is leaking

Reply:
Yeah this one I don't mind /too/ much. But still; grumble, it should use the same settings as everything else.